cpp/src/barretenberg/dsl/acir_format/acir_to_constraint_buf.hpp

```diff
@@ -180,8 +181,8 @@ void handle_blackbox_func_call(Circuit::Opcode::BlackBoxFuncCall const& arg, aci
                                           .num_bits = e.num_bits,
                                       };
                                   }),
-                    .var_message_size = arg.var_message_size.witness.value,
                     .result = map(arg.outputs, [](auto& e) { return e.value; }),
+                    .var_message_size = arg.var_message_size.witness.value,
```

Comment:
sorry, for being repetitive.... just want to make sure i mention the reordering wherever i see it